independent-publisher-connectors/WeatherForecast/apiDefinition.swagger.json

```diff
@@ -0,0 +1,229 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "weatherforecast",
+    "description": "The weather forecast connector helps one to get  weather forecast depending on the city of their choice",
+    "version": "1.0"
```

Comment:
@Haimantika The metadata is missing in your connector's swagger, please add it.

Docs - [Adding Metdata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title)